modules/component/.eslintrc.json

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "extends": [
+        "plugin:@nrwl/nx/angular",
+        "plugin:@angular-eslint/template/process-inline-templates"
+      ],
+      "parserOptions": {
+        "project": ["modules/component/tsconfig.*?.json"]
```

Comment:
Because the linter looks for `tsconfig.*?.json` config files to know which files to lint, I renamed the `tsonfig-build.json` configs to  `tsonfig.build.json`. I also could have added the `-build` variant as a project, but because we also use `tsconfig.schematics.json` and `tsconfig.spec.json`, I thought it was better to follow this notation.

Reply:
👍🏿